tests/log.sh

```diff
@@ -0,0 +1,11 @@
+source common.sh
+
+builder="$(mktemp)"
+echo -e "#!/bin/sh\nmkdir \$out" > "$builder"
+outp="$(nix-build -E \
+    'with import ./config.nix; mkDerivation { name = "fnord"; builder = '"$builder"'; }' \
+    --out-link "$(mktemp -d)/result")"
+
+test -d "$outp"
+
+nix log "$outp"
```

Comment:
Maybe merge this into tests/logging.sh? It's a bit confusing to have log.sh and logging.sh.